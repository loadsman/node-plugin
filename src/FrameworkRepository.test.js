const FrameworkRepository = require('./FrameworkRepository');

test('getExpressInfo', () => {
  expect(FrameworkRepository.getExpressInfo()).toMatchSnapshot()
})