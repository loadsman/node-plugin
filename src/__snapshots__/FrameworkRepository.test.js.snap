// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getExpressInfo 1`] = `
Object {
  "features": Array [
    "rule-list",
  ],
  "name": "Express",
  "version": "4.16.3",
}
`;
